PythonTool/GraveIdentifyAndRemove.py

```diff
@@ -1,16 +1,25 @@
+import argparse
 import numpy as np
 from pymoab import core, types, tag
 from pymoab.types import MBENTITYSET
-from sys import argv
 
 mb = core.Core()
 
-if len(argv) == 1:
-    print("WARNING: A data file was not provided.")
-    exit()
+parser = argparse.ArgumentParser(usage="Remove graveyard from a data file.")
```

Comment:
I'd suggest setting `description` instead of `usage` here. 

Argparse creates a nice print of the required and optional arguments automatically, but this is overriding it :cry:

Reply:
I second this. I think "usage" will autogenerate from the different arguments available. But you are describing what the program does, versus the usage is actually how to use it (like what the command line should look like).